automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/NotebookInteractionSpec.scala

```diff
@@ -35,33 +39,29 @@ class NotebookInteractionSpec extends FreeSpec with LeonardoTestUtils with Befor
 
   "Leonardo notebooks" - {
     val hailFileName: String = "import-hail.ipynb"
-    val hailUploadFile = ResourceFile(s"diff-tests/$hailFileName")
-
-    // must align with run-tests.sh and hub-compose-fiab.yml
-    val downloadDir = "chrome/downloads"
-    val hailDownloadFile = new File(downloadDir, hailFileName)
-    hailDownloadFile.mkdirs()
```

Comment:
This logic was a little wonky. It was creating the download file as a directory, which is wrong. It also wasn't taking the `utf8` bug into account, and it wasn't comparing it anyway due to the issue in `compareFilesExcludingIPs`. :)